src/helpers.ts

```diff
@@ -0,0 +1,19 @@
+import { Options } from "./contracts";
+
+export const CLI_ERROR_PREFIX = "CssToTs";
+export const DEFAULT_IGNORED_GLOB = "node_modules/**/*.css";

```

Comment:
Because the default `rootDir` is `.` and the tool takes all CSS files in `node_modules`. It's updated a bit, thought, just not pushed yet.